.gitignore

```diff
@@ -0,0 +1,3 @@
+.node_modules/
+test.html
+node_modules
```

Comment:
- [ ] As stated in the project requirement Kindly  Make sure that the A test.md file with the text "This file should be ignored by git" is actually ignored by git and is not present in your GitHub repository.
- 
![Capture](https://user-images.githubusercontent.com/93598189/234801359-ea716108-4d77-487a-9937-fe8f308f51e9.PNG)